retrofit/src/main/java/retrofit2/Retrofit.java

```diff
@@ -143,7 +145,11 @@
             if (platform.isDefaultMethod(method)) {
               return platform.invokeDefaultMethod(method, service, proxy, args);
             }
-            return loadServiceMethod(method).invoke(args);
+            List<Object> argsList = args != null
+                ? Arrays.asList(args)
+                : Collections.emptyList();
+            Invocation invocation = new Invocation(method, argsList);
```

Comment:
We need to push this creation farther in so that we can remove the `Continuation` argument from #2886. I don't think we want to reference that instance from the request since it could leak objects if you retain a response object for longer than we expect.

Reply:
Yeah, I was anticipating we’d strip that out, but wasn’t sure how/where. Fixed to make that easier when it lands.